hydroflow_plus_kvs_server/src/main.rs

```diff
@@ -0,0 +1,26 @@
+use hydroflow_plus::hydroflow::bytes::Bytes;
+use hydroflow_plus_kvs::my_kvs;
+use regex::Regex;
+
+fn main() {
+    let (send_0, recv_0) = hydroflow_plus::hydroflow::util::unbounded_channel::<_>();
+    let mut _other_flow = hydroflow_plus_kvs::my_example_flow!(
```

Comment:
these being a macro is kind of a bummer, their arguments aren't easily discoverable with rust analyzer.

Reply:
I guess not much can be done about it since it has to be a proc macro.

![image](https://github.com/hydro-project/hydroflow/assets/2418897/5d74730f-cbec-441d-b6dc-eafb154edab0)

and I guess the main issue is actually that the jump-to-definition stuff is broken.

Reply:
If you put the calling code in the same crate it kinda works, rust analyzer seems to confuse the macro call for a function invocation but it does display the args for you.